### PR TITLE
Update DCC dependencies

### PR DESCRIPTION
**Pull request by Marc-Andre-Rivet**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -80,7 +80,7 @@ jobs:
           command: |
             . venv/bin/activate && mkdir packages
             # build main dash
-            git clone --depth 1 https://github.com/plotly/dash.git dash-main
+            git clone -b 20200814-update-deps --depth 1 https://github.com/plotly/dash.git dash-main
```
Comment (Marc-Andre-Rivet) [inline .circleci/config.yml:83]: Temporary - to test against the new pylint & flake8 in Dash

File: .prettierrc [hunk 2]
```diff
@@ -1,6 +1,7 @@
 {
     "tabWidth": 4,
     "singleQuote": true,
+    "arrowParens": "avoid",
```
Comment (Marc-Andre-Rivet) [inline .prettierrc:4]: this default has changed..

File: .prettierrc [hunk 3]
```diff
@@ -1,6 +1,7 @@
...
     "bracketSpacing": false,
-    "trailingComma": "es5"
-}
+    "trailingComma": "none"
```
Comment (Marc-Andre-Rivet) [inline .prettierrc:6]: to be consistent with Dash settings

File: tests/test_integration_1.py [hunk 4]
```diff
@@ -389,37 +389,32 @@ def test_location_link(self):
         ])
 
         @app.callback(
-            output=Output(component_id='test-pathname',
-                          component_property='children'),
-            inputs=[Input(component_id='test-location', component_property='pathname')])
+            Output(component_id='test-pathname', component_property='children'),
```
Comment (Marc-Andre-Rivet) [inline tests/test_integration_1.py:392]: Temporary, due to Dash regression. Remove before merging.

**Comment by alexcjohnson** [comment]
superseded by #939

**Pull request closed without merging**